sklearn/covariance/graph_lasso_.py

```diff
@@ -467,7 +467,8 @@ def graphical_lasso_path(X, alphas, cov_init=None, X_test=None, mode='cd',
 
 
 class GraphicalLassoCV(GraphicalLasso):
-    """Sparse inverse covariance w/ cross-validated choice of the l1 penalty
+    """Sparse inverse covariance with :term:`cross-validated choice
```

Comment:
I don't think these references should be in the first line of a docstring. First lines have special roles in `pydoc` and elsewhere. They should be just one line, according to PEP257.

Reply:
Thanks for the feedback, would that be OK if I revert the first docstring line and add a new one saying something like:

>See glossary entry for cross-validation estimators.

(with link included)

?

Reply:
Yes.

Reply:
done